guides/source/testing.md

```diff
@@ -438,6 +444,10 @@ You can run a single test by appending a line number to a filename:
 
     bin/rails test test/models/user_test.rb:27
 
+You can run multiple test with in a line range by appending the line range to a filename:
```

Comment:
```suggestion
You can run multiple tests with in a line range by appending the line range to a filename:
```

Reply:
I overlooked this one.

Reply:
Thanks for noticing this one as well. Updated all the requested changes.